buku 4.0

zmwangx opened the pull request:
Created with `brew bump-formula-pr`.

Moisan commented:
Both `beautifulsoup4` and `certifi` resources version were lowered, is there a reason for this?

zmwangx commented:
Sorry, no, there's no reason for this. In fact, I updated these resource blocks with my semi-automated tool, but I happened to use a wrong Python and picked up a bunch of cached old versions. This is now corrected.

Moisan commented:
Thank you for the update @zmwangx !
